crates/hdk/src/hash.rs

```diff
@@ -68,6 +68,33 @@ pub fn hash_header(input: Header) -> ExternResult<HeaderHash> {
     }
 }
 
+/// Hash arbitrary bytes into an `ExternalHash`.
+///
+/// External hashes have a DHT location and hash prefix like all other native
+/// holochain hashes but are NOT found/fetchable on the DHT.
+///
+/// External hashing makes no assumptions about the data that was digested to
+/// create the hash so arbitrary bytes can be passed in. Note however that the
+/// hashing is synchronous (like everything in wasm) so avoid very large data
+/// inputs.
+///
+/// It is valid to EITHER use an existing 32 byte hash/data as literal bytes
```

Comment:
@maackle did you see this documentation for interop?

Reply:
@thedavidmeister this seems weird...I'm wondering if we shouldn't just instruct people to supply their own hash, or use one of our hashing functions to hash their data, and then use `ExternalHash::from_raw_32()`. Would you say that this function is just doing the same thing, but with the digest hardcoded to Blake2b? If so it seems like this is just slightly convenient but mainly just hides the fact that it's using Blake2b, and maybe it's better to be explicit?

Reply:
I see you took this out, which I'm in support of, but just leaving the convo open in case anyone else wants to chime in